### ISSUE TITLE
additional accessibility features for screen reader compatibility

### ISSUE THREAD

**Issue by mciethan** [bpda-research-division/neighborhood-change#59]
R Shiny doesn't have a ton of built-ins for adding alt text (useful for anyone using screen readers), but this author implemented some useful-looking functions for dynamic alt text: https://medium.com/@traffordDataLab/alt-text-for-dynamic-plots-in-shiny-2e24c684f187

Some ideas for alt-text on dropdown menus: https://stackoverflow.com/questions/71943841/alt-text-for-a-dropdown-menu-in-shiny

Some starter info on adding ARIA tags: https://stackoverflow.com/questions/70919308/how-to-implement-aria-tags-for-shiny-applications-to-meet-wcag-requiremnets

**Comment by mciethan** [comment]
for general accessibility testing there are a few options, including this package that's specific to shiny:

https://appsilon.com/r-shiny-accessibility/

**Comment by mciethan** [comment]
update: I tried using a screen reader, and found that the tab selections and drop-down menu selections are actually OK to navigate, although the whole experience is definitely clunkier. It also was able to read out the map and the download button. However, I was unable to interact with the slider, and there was no narration for the charts. The solution for dynamic alt text linked above is unfortunately limited to renderPlot() calls and does not seem to expand to renderPlotly(), unless this feature request (https://github.com/plotly/plotly.R/issues/2106) makes any traction.
